stac_ipyleaflet/stac_discovery/stac_widget.py

```diff
@@ -93,16 +169,22 @@ def template(self) -> Box( style={"max_height: 200px"}):
         )
         collection_description_box = VBox(
             [
-                HTML(value="<b>Description</b>", style=styles["init"], layout=layouts["header"],),
-                collection_description
+                HTML(
+                    value="<b>Description</b>",
+                    style=styles["init"],
+                    layout=layouts["header"],
+                ),
+                collection_description,
             ]
         )
         collection_url = HTML(
             value=f'<a href={self.stac_data["collection"]["href"]} target="_blank">{self.stac_data["collection"]["href"]}</a>',
             style=styles["init"],
             layout=layouts["default"],
         )
-        stac_browser_url = self.stac_data["collection"]["href"].replace("https://", "https://stac-browser.maap-project.org/external/")
+        stac_browser_url = self.stac_data["collection"]["href"].replace(
+            "https://", "https://stac-browser.maap-project.org/external/"
+        )
```

Comment:
💭  : I'm realizing that alot of these changes are probably due to the black formatter lol. But i'm thinking `https://stac-browser.maap-project.org/external/` is a good candidate as a constant